app/views/DR/ReportScreen/sections/SetpAdress.js

```diff
@@ -46,6 +48,23 @@ const StepAdress = ({ setCompleted }) => {
     });
     setMunicipios(municipNames);
   };
+
+  const getCurrentLocation = async () => {
+    const granted = await PermissionsAndroid.check(
+      PermissionsAndroid.PERMISSIONS.ACCESS_FINE_LOCATION,
+    );
+    if (granted || Platform.OS === 'ios') {
+      Geolocation.getCurrentPosition(
+        ({ coords }) => {
+          setSelectedOption('latitude', coords.latitude.toString());
+          setSelectedOption('longitude', coords.longitude.toString());
+        },
+        () => {},
```

Comment:
Why this arrow function?

Reply:
I miss it when cleaning. Will remove it now

Reply:
I missed it when cleaning, will remove it now